### DISCUSSION TITLE
Is Budibase propietary?

### DISCUSSION THREAD

**Discussion by Miguelklappes** [Budibase/budibase#970] [Help] [answered]
Hello to all,

What I mean with this question is, can I export the generated code? Or is a lock-in kind of app (like that thing called Bubble).

Thanks!

**Comment by joebudi** [comment, marked as answer]
Hi @Miguelklappes 

Thank you for reaching out.

Budibase is open-source - further info regarding licensing is here: https://github.com/Budibase/budibase#-license
You will be able to take your apps, and self-host very soon. This is different from Bubble.

For further information on self-hosting, check out this [discussion](https://github.com/Budibase/budibase/discussions/648) and follow the progress of self-hosting [here](https://github.com/Budibase/budibase/projects/10).

Joe
